src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,73 @@
-import React from 'react';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+const PICTURE = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+interface Props {
+  onAddMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [isLoaded, setIsLoading] = useState(false);
+
+  const handlerSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    const isValidQuery = query && query.trim().length;
+
+    const movieFromServer = await getMovie(query);
+
+    if ('Error' in movieFromServer || !isValidQuery) {
+      setIsError(true);
+      setIsLoading(false);
+
+      return;
+    }
+
+    const {
+      Title,
+      Plot,
+      imdbID,
+      Poster,
+    } = movieFromServer;
+
+    setMovie({
+      title: Title,
+      description: Plot,
+      imgUrl: Poster !== 'N/A'
```

Comment:
Create a separate variable for it
```suggestion
      imgUrl,
```